examples/README.md

```diff
@@ -1,4 +1,68 @@
-# Doorman Integration Examples
+# Integration Examples
 
 - [Python / Flask](python/): A Web UI interacts with Auth0 and a Flask API
 
+# Configuration Examples
+
+## Superusers with Doorman tags
+
+We define a tag ``superuser`` and use it in principals in policies rules.
+
+```yaml
+
+    service: https://api.service.org
+    tags:
+      superuser:
+        - userid:maria
+        - group:admins
+    policies:
+      -
+        id: super-users
+        description: Superusers can do everything
+        principals:
+          - tag:superuser
+        actions:
+          - <.*>
+        resources:
+          - <.*>
+```
```

Comment:
So this is an example where you list user IDs, right? Effectively this is a yaml-talk for saying that `maria` has the `superuser` tag in Doorman?

I don't understand the `group:admins` example. I can only speculate that it means that people who are in the "admins" group (where is that defined?!) also get the `superuser` tag.